### PR TITLE
slo-controller report koordinator-gpu resource dimension

### PR DESCRIPTION
**Pull request by buptcozy**
Signed-off-by: yangzhang <bupt_cozy@126.com>

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/slo-controller/noderesource/noderesource.go [hunk 1]
```diff
@@ -182,6 +185,8 @@ func (r *NodeResourceReconciler) updateGPUNodeResource(node *corev1.Node, device
 		updateNode.Status.Allocatable[extension.GPUCore] = *coreTotal
 		updateNode.Status.Capacity[extension.GPUMemoryRatio] = *ratioTotal
 		updateNode.Status.Allocatable[extension.GPUMemoryRatio] = *ratioTotal
+		updateNode.Status.Capacity[extension.KoordGPU] = *koordGpuTotal
```
Comment (zwzhang0107) [inline pkg/slo-controller/noderesource/noderesource.go:188]: how about add some ut for this

**Comment by ZYecho** [comment]
LGTM

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

File: pkg/slo-controller/noderesource/noderesource.go (hunk 1 above)
Comment (buptcozy) [reply pkg/slo-controller/noderesource/noderesource.go:188]: the exist logic has no ut yet

**Review by jasonliu747 — APPROVED** [review]

**Review by yihuifeng — APPROVED** [review]
/lgtm

**Pull request merged**